src/status_im/contexts/profile/settings/screens/password/change_password/effects.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.contexts.profile.settings.screens.password.change-password.effects
+  (:require [native-module.core :as native-module]
+            [promesa.core :as promesa]
+            [status-im.common.keychain.events :as keychain]
+            [utils.re-frame :as rf]
+            [utils.security.core :as security]))
+
+(rf/reg-fx
+ :effects.change-password/change-password
+ (fn [{:keys [key-uid old-password new-password on-success on-fail]}]
+   (let [hash-masked-password (fn [pw]
+                                (-> pw security/safe-unmask-data native-module/sha3))
```

Comment:
Can we extract this function into a static function?

Reply:
I added one a while back to the security ns but forgot about it :brain: . Will use it here and in the case below